computervision/src/main/java/ComputerVisionInAction.kt

```diff
@@ -16,23 +17,28 @@ internal class ComputerVisionInAction(private val cvPortal: SensorPortal) : Comp
     }
 
     private fun cvTokenize(rawData: String): Vision? {
-        val raw = JSONObject(rawData)
-        return when (raw.getInt("type") == 1) {
-
-            raw.has("value") -> Vision.OCR(raw.optString("value"))
-            raw.has("value") -> Vision.Scenery(raw.optString("value"))
-            raw.has("value") -> Vision.Facial(raw.optString("value"))
-            raw.has("value") -> Vision.Emotion(raw.optString("value"))
-            raw.has("value") -> Vision.ObjectDetection(raw.optJSONArray("value"))
-            else -> null // (corrupt reading, discard.) the russians did it for cv !
+        try {
+            val raw = JSONObject(rawData)
+            val list: List<Any> = listOf<Any>(raw.getJSONArray("value")) //[[......]]
```

Comment:
This list should be only retrieved when it is object detection.

Reply:
Indeed it should be :''D
🙇‍♀ 🙇‍♀